README.md

```diff
@@ -2,6 +2,8 @@
 
 This repository is a collection of software and documentation templates for the Backstage Orchestrator plugin. The templates are organized into two folders: `scaffolder-templates` for Software Templates and `documentation-templates` for general documentation templates. Contributions are welcome for new and existing templates!
 
+Starting RHDH orchestrator plugin v1.13, the outputs schema is used to improve how the outputs of a workflow are displayed to the user; instead of displaying a JSON of the whole context, only the defined field will be formatted and displayed in an user-friendly way.
+
```

Comment:
WDYT of using friend's recommendation?

> Starting from RHDH Orchestrator Plugin v1.13, the outputs schema enhances the display of workflow outputs. Rather than showing the entire context as a JSON object, only the specified fields are formatted and presented in a more user-friendly manner.

Reply:
also, pls include a link to the schema definition and description:
https://github.com/parodos-dev/serverless-workflows/blob/main/best-practices.md#workflow-output-schema